colorama/tests/initialise_test.py

```diff
@@ -75,13 +74,16 @@ def testInitWrapOffDoesntWrapOnWindows(self):
     def testInitWrapOffIncompatibleWithAutoresetOn(self):
         self.assertRaises(ValueError, lambda: init(autoreset=True, wrap=False))
 
-    @patch('colorama.ansitowin32.winterm', None)
     @patch('colorama.ansitowin32.winapi_test', lambda *_: True)
-    def testInitOnlyWrapsOnce(self):
-        with osname("nt"):
+    def testInitTwiceCanUndoneWithDeinitOnce(self):
+        with osname('nt'):
+            self.assertNotWrapped()
             init()
+            self.assertWrapped()
             init()
             self.assertWrapped()
+            deinit()
+            self.assertNotWrapped()
```

Comment:
Maybe we should add another "init" to make sure init->deinit->init works

Reply:
Yes, I have arrived at the same conclusion! :-) Doing...